python/sbp/RELEASE-VERSION

```diff
@@ -1 +1 @@
-4.3.1-alpha
\ No newline at end of file
+4.3.1
```

Comment:
Should drop this change, anything that's not a released version needs to maintain the `-alpha` tag.

Reply:
should i update that to 4.3.2-alpha instead of 4.3.1, since i deployed it as 4.3.2, (couldnt deploy as 4.3.1 for some reason)

Reply:
After release there is supposed to be a "prep for next release" process which bumps this to `<next_bugfix_version>-alpha` -- so if 4.3.2 was just released, this should be `4.3.3-alpha` -- this happens automatically if you rerun the generator on a non-tagged commit, e.g. https://github.com/swift-nav/libsbp/commit/b423ed3e487c8a6890dc57ef842989ed3ac58e10